policy/datasets.yaml

```diff
@@ -606,6 +620,9 @@ dcrCoadd_det_schema:
     python: lsst.afw.table.SourceCatalog
     template: schema/dcrCoadd_det.fits
 dcrCoadd_mergeDet:
+    description: >
```

Comment:
Is it too late to object to this name and rename it `dcrCoadd_mergeDetections`?

Reply:
I think so. The *Coadd_mergeDet name is used in many other places, and I'm just following the existing pattern.